src/index.ts

```diff
@@ -388,25 +356,34 @@ export async function cli(args: string[]) {
   }
 
   const pkgManifest = require(path.join(cwd, 'package.json'));
+  const implicitDependencies = [
+    ...Object.keys(pkgManifest.dependencies || {}),
+    ...Object.keys(pkgManifest.peerDependencies || {}),
+  ];
+  const allDependencies = [
+    ...Object.keys(pkgManifest.dependencies || {}),
+    ...Object.keys(pkgManifest.peerDependencies || {}),
+    ...Object.keys(pkgManifest.devDependencies || {}),
+  ];
+
+  let isExplicit = false;
+  const installTargets = [];
   const {namedExports, webDependencies, dedupe} = pkgManifest['@pika/web'] || {
     namedExports: undefined,
     webDependencies: undefined,
     dedupe: undefined,
   };
 
-  let doesWhitelistExist = true;
-  const arrayOfDeps = [...webDependencies];
+  if (webDependencies) {
+    isExplicit = true;
+    installTargets.push(...scanWhitelist(webDependencies, cwd));
```

Comment:
Minor inclusivity suggestion: it’d be nice to avoid the terms “whitelist” / “blacklist” if possible. While it’s contested, and I think the meaning is clear here, if there’s another word that fits the bill I’d like to use that.

Reply:
Bah, thanks for catching that and saying something

Reply:
renamed to "Dep List" or "Dependencies List"